Libplanet.Tools/ConsolePasswordReader.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Text;
+
+namespace Libplanet.Tools
+{
+    public class ConsolePasswordReader
+    {
+        public static string Read(string prompt)
+        {
+            var password = new StringBuilder();
+
+            Console.Write(prompt);
+            do
+            {
+                var key = Console.ReadKey(true).KeyChar;
+
+                if (key == '\b')
+                {
+                    if (password.Length <= 0)
+                    {
+                        continue;
+                    }
+
+                    password.Remove(password.Length - 1, 1);
+                    Console.SetCursorPosition(Console.CursorLeft - 1, Console.CursorTop);
+                    Console.Write(' ');
+                    Console.SetCursorPosition(Console.CursorLeft - 1, Console.CursorTop);
+                }
+                else if (key == '\r')
+                {
+                    Console.WriteLine();
+                    break;
+                }
+                else if (!char.IsControl(key))
+                {
+                    password.Append(key);
+                    Console.Write('*');
+                }
+            }
+            while (true);
+
+            Console.WriteLine(password.ToString());
```

Comment:
It seems expose `password` via `Console`. is it intended?

Reply:
Apparently it's unintended…

Reply:
Sorry for unintended debug code. I removed it.